.rubocop.yml

```diff
@@ -43,6 +43,9 @@ Style/IfInsideElse:
 Style/RedundantReturn:
   Enabled: false
 
+Style/WordArray:
+  MinSize: 5
```

Comment:
That reasoning seems sound to me - was just curious.  I say you check this in, and if it causes us pain we can adjust it down the road.  I agree that this seems like a strict improvement on not enforcing anything.

Thanks!